.github/workflows/publish.yml

```
@@ -28,14 +28,21 @@ jobs:
       run: |
         python -m pip install --upgrade pip
         python -m pip install build
+        python -m pip install twine
 
     - name: Build packages
       run: python -m build
 
+    - name: Check metadata verification
+      run: python -m twine check dist/*
```

```suggestion
      run: python -m twine check --strict dist/*
```

to [turn warnings into errors](https://twine.readthedocs.io/en/stable/#twine-check)